chain/beacon/store.go

```diff
@@ -113,13 +113,20 @@ func (d *discrepancyStore) Put(b *chain.Beacon) error {
 	if err := d.Store.Put(b); err != nil {
 		return err
 	}
+
+	beaconID := d.group.ID
+	if beaconID == "" {
+		beaconID = "default"
```

Comment:
Please use the global variable for the default beacon id.

Reply:
sorry, my fingers were faster than my brain. Fixed.